types/n3/index.d.ts

```diff
@@ -130,9 +130,10 @@ export interface BlankTriple<Q extends RDF.BaseQuad = RDF.Quad> {
 
 export interface ParserOptions {
     format?: string;
-    prefixes?: string[];
+    prefixes?: { [prefix: string]: string };
```

Comment:
As Joachim has said, this appears to be a mistake in the typings, as `prefixes` shouldn't be here at all.